modAionImpl/src/org/aion/zero/impl/db/AionRepositoryImpl.java

```diff
@@ -512,10 +512,6 @@ public void flush() {
                         db.commit();
                     }
                 }
-            } else {
```

Comment:
What scenario we will see the databaseGroup equal to null ? Sometimes We can see this message during the seednode running, also the project test case.

Reply:
The databaseGroup is null when using a repository snapshot. The snapshot is always used when importing a side chain block. The message is really useless, cause it doesn't notify us of anything meaningful.